js/client/modules/@arangodb/testsuites/server_permissions.js

```diff
@@ -59,26 +60,37 @@ const testPaths = {
 class permissionsRunner extends tu.runLocalInArangoshRunner {
   constructor(options, testname, ...optionalArgs) {
     super(options, testname, ...optionalArgs);
+    this.options.extremeVerbosity = true;
     this.info = "runInDriverTest";
   }
-  run(testlist) {
+  run(testList) {
+    let beforeStart = time();
+    this.continueTesting = true;
+    this.testList = testList;
+    let testrunStart = time();
+    this.results = {
+      shutdown: true,
+      startupTime: testrunStart - beforeStart
```

Comment:
Not sure if this number is meaningful.

Reply:
yes, I wanted to make it look as similar as possible to the original `run` function it overrules